emcc.py

```diff
@@ -2195,11 +2195,11 @@ def compile_source_file(i, input_file):
           assert len(input_files) == 1
           input_file = input_files[0][1]
           temp_file = temp_files[0][1]
-          bitcode_target = specified_target if specified_target else unsuffixed_basename(input_file) + final_suffix
```

Comment:
It `else` block we are in here mean that `specified_target` is always truthy here.